### ISSUE TITLE
Allow specifying explicit types to bind

### ISSUE THREAD

**Issue by tritao** [mono/Embeddinator-4000#14]
As suggested by @jonpryor, allow specifying explicit types to bind with the CLI interface.

Maybe this could be done with either a whitelist or blacklist filter command line option, with wildcard support.

**Comment by realvictorprm** [comment]
Something like TypeMaps in CppSharp @tritao?

**Comment by tritao** [comment]
No, just a command line option, say `-filter MyNamespace:*` which would only allow types from that namespace to be bound.
